Data Structures/Singly Circular Linked List/Node.cpp

```diff
@@ -0,0 +1,28 @@
+// CNODE PROGRAM CLASS "NODE"
+
+class Node
+{
+	int value;
+	Node *nextnode;			// POINTER OF SAME CLASS TYPE

```

Comment:
if possible, please do start commenting with how this particular block of code would affect the overall execution of the code.